cores/arduino/stm32/usb/usbd_desc.c

```diff
@@ -225,38 +215,6 @@ uint8_t *USBD_SerialStrDescriptor(USBD_SpeedTypeDef speed, uint16_t *length)
   return (uint8_t *)USBD_StringSerial;
 }
 
-/**
-  * @brief  Returns the configuration string descriptor.
-  * @param  speed: Current device speed
-  * @param  length: Pointer to data length variable
-  * @retval Pointer to descriptor buffer
-  */
-uint8_t *USBD_Class_ConfigStrDescriptor(USBD_SpeedTypeDef speed, uint16_t *length)

```

Comment:
Oh good point, the configuration index was used somewhere without an explicit constant apparently.

I wanted to remove this to simplify the string configuration a bit. I can still imagine it makes sense to remove these, since having a string descriptor for the configuration or interface when there is only one does not really add something. When there are multiple (e.g. to distinguish between DFU and CDC interfaces, or HID keyboard and mouse), then it would indeed make sense to assign descriptors, but then you would need more than one, so the current callback-based approach that allows just a single string per type, is not really useful either (instead we would need to enable the `USBD_SUPPORT_USER_STRING_DESC` define and use the related callback along with some management of string indices, but I wanted to leave that for later).

Reply:
I agree this is not really relevant, anyway, I would not remove them to kept it as example.
It should be better to be able to define those String, like this anyone could define it like you do for the manufacturer.

Reply:
Ok, I dropped that commit, and just left these strings as they were (I did not add machinery to allow changing them from the board files, since that does not seem really relevant to me, we can always add them when someone needs that).